evaluation/swe_bench/scripts/run_infer.sh

```diff
@@ -16,6 +16,14 @@ echo "AGENT: $AGENT"
 echo "AGENT_VERSION: $AGENT_VERSION"
 echo "MODEL_CONFIG: $MODEL_CONFIG"
 
+# Default to NOT use Hint
+export USE_HINT_TEXT=false
```

Comment:
Not sure if we should set it to true or false by default... let's see... I feel like we probably would want to set it to true by default

Reply:
Yet! I tweak to to true after our on slack discussion